docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -334,41 +346,42 @@ public class CustomTenantResolver implements TenantResolver {
 ----
 ====
 
-== Starting and Configuring the Keycloak Server
+== Starting and configuring the Keycloak server
 
-To start a Keycloak Server you can use Docker and just run the following command:
+To start a Keycloak server, you can use Docker and run the following command:
 
 [source,bash,subs=attributes+]
 ----
 docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8180:8080 quay.io/keycloak/keycloak:{keycloak.version} start-dev
 ----
 
-where `keycloak.version` should be set to `17.0.0` or higher.
+Where `keycloak.version` is set to `23.0.0` or higher.
```

Comment:
IMHO `where` works better, as it continues the command. With capital `Works` the sentence looks standalone and reads unusual

Reply:
Updated. Thanks.